client/src/components/LslrSection.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class='section'>
+    <div class='header-section'>
+      <div class='h1-header-large'>{{ messages.LSLR_HEADER }}</div>
+      <div class='h2-header-large'>{{ messages.LSLR_SUBHEADER }}</div>
+      <button class='gold-button'>
+        <a :href='cityLink' target='_blank' rel='noopener noreferrer'>
+          Go to city of {{ city }} website
+        </a>
+      </button>
+    </div>
+  </div>
+</template>
+
+<script lang='ts'>
+import { defineComponent, PropType } from 'vue';
+import { ScorecardMessages as messages } from '@/assets/messages/scorecard_messages';
+import { City } from '@/model/states/model/geo_data';
+
+export const LSLR_CITY_LINKS: Map<City, string> = new Map<City, string>([
+  [City.toledo, 'https://toledo.oh.gov/residents/water/lead-service-lines/customer-side'],
+]);
+
+/**
+ * Lslr section component.
```

Comment:
nit: could you define what LSLR means here or somewhere?

Reply:
Done!